kiliautoml/utils/pytorchvision/trainer.py

```diff
@@ -34,14 +37,15 @@ def train_model_pytorch(
     optimizer = optim.SGD(model.parameters(), lr=0.001, momentum=0.9)
 
     model = model.to(device)
-    dataset_sizes = {x: len(dataloaders[x].dataset) for x in ["train", "val"]}
-
     # Decay LR by a factor of 0.1 every 7 epochs
     scheduler = lr_scheduler.StepLR(optimizer, step_size=7, gamma=0.1)
 
     best_model_wts = copy.deepcopy(model.state_dict())
-    best_acc = 0.0
-    best_loss = float("inf")
+    best_val_acc = epoch_train_acc = corresponding_train_acc = 0.0
+    best_val_precision = best_val_recall = best_val_f1 = []
```

Comment:
won't work, this will make `best_val_precision, best_val_recall,  best_val_f1` have the same values since they are pointing towards the same list. Same below